src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -26,12 +68,18 @@ const DeleteWorkspaceModal = ({ workspace: { workspace: { namespace, name, bucke
     }
   }
 
+  const isDeleteDisabledFromApps = hasApps() && !_.isEmpty(nonDeletableApps)
+
   return h(Modal, {
     title: 'Delete workspace',
     onDismiss,
     okButton: h(ButtonPrimary, {
-      disabled: _.toLower(deleteConfirmation) !== 'delete workspace',
-      onClick: () => deleteWorkspace()
+      disabled: _.toLower(deleteConfirmation) !== 'delete workspace' || isDeleteDisabledFromApps,
+      onClick: () => deleteWorkspace(),
+      tooltip: Utils.cond(
+        [_.toLower(deleteConfirmation) !== 'delete workspace', () => 'You must type the confirmation message'],
```

Comment:
I'm seeing the message "You must type the confirmation message" when I have an App in `precreating`, I think you want to flip these cond checks so the App one is first